test/unit/CameraUtils_unit_test.js

```diff
@@ -0,0 +1,135 @@
+/* global describe, it */
+import * as THREE from 'three';
+import assert from 'assert';
+import Coordinates, { ellipsoidSizes } from '../../src/Core/Geographic/Coordinates';
+import Ellipsoid from '../../src/Core//Math/Ellipsoid';
+import CameraUtils from '../../src/utils/CameraUtils';
+import DEMUtils from '../../src/utils/DEMUtils';
+
+THREE.Object3D.DefaultUp = new THREE.Vector3(0, 0, 1);
+
+DEMUtils.getElevationValueAt = () => {
+    const result = { z: 0 };
+    return result;
+};
+
+const raycaster = new THREE.Raycaster();
+const center = new THREE.Vector2();
+const ellipsoid = new Ellipsoid(ellipsoidSizes());
+
+function pickEllipsoid(camera) {
+    raycaster.setFromCamera(center, camera);
+    return ellipsoid.intersection(raycaster.ray);
+}
+
+const view = {};
+const camera = new THREE.PerspectiveCamera();
+
+view.getPickingPositionFromDepth = function getPickingPositionFromDepth() {
+    return pickEllipsoid(camera);
```

Comment:
I must to overload the function `getPickingPositionFromDepth` because it doesn't work properly. Because Chrome in headless mode doesn't support the WebGL EXT_frag_depth extension.

Reply:
Weird, but ok